config/internal/apiserver/sample-pipeline/sample-config.yaml.tmpl

```diff
@@ -7,11 +7,21 @@ metadata:
         app: {{.APIServerDefaultResourceName}}
         component: data-science-pipelines
 data:
-    sample_config.json: |-
-        [
-          {
-            "name": "[Demo] iris-training",
-            "description": "[source code](https://github.com/opendatahub-io/data-science-pipelines/tree/master/samples/iris-sklearn) A simple pipeline to demonstrate a basic ML Training workflow",
-            "file": "/samples/iris-pipeline-compiled.yaml"
-          }
-        ]
+  sample_config.json: |-
+    [
+      {{- if .EnableSamplePipeline }}
+      {
+        "name": "[Demo] iris-training",
+        "description": "[source code](https://github.com/opendatahub-io/data-science-pipelines/tree/master/samples/iris-sklearn) A simple pipeline to demonstrate a basic ML Training workflow",
+        "file": "/samples/iris-pipeline-compiled.yaml"
+      }
+      {{- if .EnableIlabKFPPipeline }},{{ end }}
+      {{- end }}
+      {{- if .EnableIlabKFPPipeline }}
+      {
+        "name": "ilab-kfp-pipeline",
```

Comment:
This is a human-readable name so lets give it a friendlier title, like `[Instructlab] Multi-Phase Training Pipeline`